shell/src/main/java/alluxio/cli/bundler/command/AbstractCollectInfoCommand.java

```diff
@@ -56,12 +56,15 @@ public void validateArgs(CommandLine cl) throws InvalidArgumentException {
    * */
   public String getWorkingDirectory(CommandLine cl) {
     String[] args = cl.getArgs();
-    String baseDirPath = args[0];
+    String baseDirPath = args[1];
     String workingDirPath =  Paths.get(baseDirPath, this.getCommandName()).toString();
     LOG.debug("Command %s works in %s", this.getCommandName(), workingDirPath);
     // mkdirs checks existence of the path
     File workingDir = new File(workingDirPath);
-    workingDir.mkdirs();
+    if (!workingDir.exists()) {
+      System.out.format("Working path %s does not exist. mkdir first%n", workingDirPath);
```

Comment:
This message makes it sound like the user should run mkdir. Instead, this could say:

"Creating working directory: %s %n"

So it sounds like the command is creating the directory.